src/main/java/ai/toloka/client/v1/impl/WebhookSubscriptionClientImpl.java

```diff
@@ -70,4 +72,12 @@ public WebhookSubscription getWebhookSubscription(String webhookSubscriptionId)
     public void deleteWebhookSubscription(String webhookSubscriptionId) {
         delete(webhookSubscriptionId, WEBHOOK_SUBSCRIPTIONS_PATH);
     }
+
+    @Override
+    public ModificationResult<WebhookPushResult> sendTestNotification(String webhookSubscriptionId) {
```

Comment:
lets rewrite to `sendTestWebhook`, using notification and webhook both terms may be confusing

use 'test notification' instead of 'test webhook'